perllib/FixMyStreet/App/Form/Hercules/UPRN.pm

```diff
@@ -0,0 +1,36 @@
+package FixMyStreet::App::Form::Hercules::UPRN;
+
+use utf8;
+use HTML::FormHandler::Moose;
+extends 'HTML::FormHandler';
+
+use mySociety::PostcodeUtil qw(is_valid_postcode);
+
+has '+field_name_space' => ( default => 'FixMyStreet::App::Form::Field' );
+
+has cobrand => ( is => 'ro' );
+
+has_field postcode => (
+    required => 1,
```

Comment:
Not sure if it's possible here or elsewhere but the postcode input doesn't get focus when the page loads which I suspect it should.